scripts/install

```diff
@@ -53,7 +53,7 @@ sed -i "s@YNH_ADMIN_USER@$admin@g" ../conf/dokuwiki.php
 sudo mkdir "$final_path"
 ynh_app_setting_set $app final_path $final_path
 
-SETUP_SOURCE "dokuwiki-c5525093cf2c4f47e2e5d2439fe13964.tgz"
+SETUP_SOURCE "dokuwiki-stable.tgz"
```

Comment:
The source URL is in the sources/source_url file, it does not need to be changed for that. install and upgrade script use a fonction SETUP_SOURCE for download a new source.